core/src/main/java/org/infinispan/container/offheap/BoundedOffHeapDataContainer.java

```diff
@@ -260,7 +260,7 @@ private void ensureSize() {
                InternalCacheEntry<WrappedBytes, WrappedBytes> ice = offHeapEntryFactory.fromMemory(addressToRemove);
                passivator.passivate(ice);
                // TODO: this rereads the object from memory again!
```

Comment:
@wburns Is this TODO still relevant?

Reply:
Oops thought I removed that, let me remove just that line real quick